packages/dashboard/src/scss/main.scss

```diff
@@ -10,6 +10,11 @@
     margin-top: 20%;
   }
 
+  > .cove-editor__content {
+    width: 100% !important;
```

Comment:
I've seen width: 100% !important; under another class declaration that had .cove-editor__content